src/codegen/class.ts

```diff
@@ -112,7 +112,7 @@ export async function* generateClasses(
   yield `import { LanguageTag, parseLanguageTag }
     from "@phensley/language-tag";\n`;
   yield `import { decode as decodeMultibase, encode as encodeMultibase }
-    from "multibase";`;
+    from "../multibase/index.ts";`;
```

Comment:
Even though it already works, could you move the *src/multibase* directory to *src/runtime/multibase* for consistency? All local code that the generated code depends on should be inside *src/runtime/*:

```suggestion
    from "${runtimePath}/multibase/index.ts";`;
```